packages/rpc-core/src/rpc-methods/getEpochSchedule.ts

```diff
@@ -16,6 +16,10 @@ type GetEpochScheduleApiResponse = Readonly<{
 export interface GetEpochScheduleApi {
     /**
      * Returns the epoch schedule information from this cluster's genesis config
+     * Note that the optional config object is ignored. See https://github.com/solana-labs/solana-web3.js/issues/1389
      */
-    getEpochSchedule(): GetEpochScheduleApiResponse;
+    getEpochSchedule(
+        // FIXME: https://github.com/solana-labs/solana-web3.js/issues/1389
+        config?: Record<string, never>
```

Comment:
_Maybe_ consider naming this argument something else? Think of what you'd want to show up in autocomplete.

```ts
// Is this the worst idea?
getEpochSchedule(NO_CONFIG?: Record<string, never>)
```